### DISCUSSION TITLE
How to export all entries/elements?

### DISCUSSION THREAD

**Discussion by janhenckens** [craftcms/cms#13736] [Q&A] [answered]
When using Craft's own exporter, we can select all visible item on the page and export those - but is there a way to export **all** elements of type you're viewing at that time?

Use-case here is exporting Formie submissions, which the client now does per page and then they merge the files themselves.

**Comment by brandonkelly** [comment, marked as answer]
Exporting all matching entries all at once would risk exceeding the system’s resource limits.

We should definitely make it possible, but it will require a queue-based approach where the resulting export (report) would be generated in batches, and you’d get notified once it’s ready to be downloaded.

**Reply by janhenckens** [reply]
Got it, that would indeed be problematic.

(that's how we've been doing our custom exports now, through a queue job and with a notification to the user that requested the export).
